TESTS/mbed_hal/rtc_time/main.cpp

```diff
@@ -53,6 +51,7 @@ bool is_leap_year(int year)
  * When _rtc_is_leap_year() function is called.
  * Then _rtc_is_leap_year() returns true if given year is a leap year; false otherwise.
  */
+template <rtc_leap_year_support_t rtc_leap_year_support>
```

Comment:
Why is this defined twice?

Reply:
Lines 54 and 93

Reply:
Because these are two different template functions. Is it possible to define one template for many functions? I have never seen such case.

Reply:
It just seemed odd to me, but our local C++ expert (@geky) mentioend that it's par for the course. I'm use to seeing templates be used in classes, not functions.